ansible/roles/cri/tasks/crictl.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: 'crictl | download crictl binary from kubernetes-sig release repository'
+  ansible.builtin.get_url:
+    url: '{{ cri_crictl_download_url }}'
+    dest: '/tmp/crictl.tar.gz'
+    mode: 0440
+
+- name: 'crictl | unarchive crictl binary'
+  ansible.builtin.unarchive:
+    src: '/tmp/crictl.tar.gz'
+    dest: '/usr/local/bin'
+    remote_src: true
+
+- name: 'crictl | ensure crictl config file is up to date'
+  ansible.builtin.template:
+    src: 'crictl.yaml.j2'
+    dest: '/etc/crictl.yaml'
+
+- name: 'crictl | set crictl shell completion'
+  ansible.builtin.command: |
+    /usr/local/bin/crictl completion
+  changed_when: false
```

Comment:
As is was discussed out side of github I would strongly prefer to have containerd managed by the distro's package manager. Currently log2ram is only outlier. 

If we're okay with providing this through package managers we can ship this now and revise on our code cleanup.